front/app/modules/commercial/events_widget/index.tsx

```diff
@@ -42,7 +44,7 @@ const configuration: ModuleConfiguration = {
     'app.containers.HomePage.EventsWidget': () => {
       return (
         <RenderOnAllowedAndEnabled>
-          <EventsWidget />
+          <EventsWidget id="e2e-homepage-events-widget-container" />
```

Comment:
In the other test the selector is ` cy.get('[data-testid="e2e-homepage-events-widget-container"]')` which would be for `<EventsWidget data-testid="e2e-homepage-events-widget-container" />`

For e2e tests you want `data-cy="e2e-homepage-events-widget-container"` and ` cy.get('[data-cy="e2e...`

Reply:
Thanks, changed!